integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -675,12 +675,17 @@ public void ttlRenameTest() throws Exception {
     CreateOptions options = new CreateOptions.Builder(MasterContext.getConf()).setTTL(ttl).build();
     long fileId = mFsMaster.create(new TachyonURI("/testFolder/testFile1"), options);
     mFsMaster.renameInternal(fileId, new TachyonURI("/testFolder/testFile2"), true,
-        System.currentTimeMillis());
+        getTimeMs());
     FileInfo folderInfo =
         mFsMaster.getFileInfo(mFsMaster.getFileId(new TachyonURI("/testFolder/testFile2")));
     Assert.assertEquals(ttl, folderInfo.ttl);
   }
 
+  private long getTimeMs() {
+    // Modify current time to make sure setOperationTimeMs is respected
+    return System.currentTimeMillis() - 1000;
```

Comment:
it's an arbitrary number that will make it unlikely for an implementation which ignores the specified opTime to pass the unit test. Other values to avoid could be `0`, `System.currentTimeMillis() + 1`, `System.currentTimeMillis()`.

Comment has been improved.